### PR TITLE
audit: fix deepcopy registration

### PR DESCRIPTION
**Pull request by sttts**
Remove manual registration into scheme by fixing the deepcopy tag of the api group.

### CODE REVIEW

**Comment by sttts** [comment]
@smarterclayton approved?

File: staging/src/k8s.io/apiserver/pkg/apis/audit/register.go [hunk 1]
```diff
@@ -49,5 +49,6 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Policy{},
 		&PolicyList{},
 	)
+	scheme.AddGeneratedDeepCopyFuncs(GetGeneratedDeepCopyFuncs()...)
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/register.go]: Note: this ugly line goes away with phase 2 of static deepcopy https://github.com/kubernetes/kubernetes/pull/48544

**Comment by timstclair** [comment]
/lgtm

Thanks!

**Pull request merged**